AutomationPractice/features/Pages/AuthencationPage.py

```diff
@@ -1,15 +1,19 @@
 import Locator
+from features.browsers.browser import Browser
 
 
-class Authencation():
+class Authencation(Browser):
     def __init__(self, driver):
         self.driver = driver
 
     def getCreateEmailFiled(self):
```

Comment:
please follow naming convention. Method name should be lowercases

Reply:
fixed as aboved comment